LastWork(newWork)/css/p_evaluations.css

```diff
@@ -0,0 +1,81 @@
+.image{
+	width: 150px;
+	height: 30px;
+	margin-left: 50px;
+}
+
+.header{
+	width: 100%;
+	height: 55px;
+	border-bottom: 1px solid;
+	display: inline-flex;
+	align-items: center;
+	box-shadow: 0px 3px 10px grey;
+}
+
+.header_div{
```

Comment:
Do not include the name of the DOM element in the selectors. Classes can be applied to other elements besides div